src/Storage/FileStorage.php

```diff
@@ -143,4 +142,21 @@ private function gc(): void
             }
         }
     }
+
+    /**
+     * @return string[]
+     */
+    private function findFilesOrderedByModifiedTime(string $pattern): array
+    {
+        $files = glob($pattern, GLOB_NOSORT);
+        if ($files === false) {
+            return [];
+        }
+
+        usort(
```

Comment:
Is it ok it's changed from `uasort`? Looks like the result keys aren't used here, but anyway

Reply:
Yes, keys never used.